go.mod

```diff
@@ -8,17 +8,34 @@ require (
 )
 
 require (
+	github.com/Masterminds/goutils v1.1.0 // indirect
+	github.com/Masterminds/semver v1.5.0 // indirect
+	github.com/Masterminds/sprig v2.22.0+incompatible
+	github.com/araddon/dateparse v0.0.0-20200409225146-d820a6159ab1
+	github.com/ghodss/yaml v1.0.0
 	github.com/globalsign/mgo v0.0.0-20181015135952-eeefdecb41b8
 	github.com/gofrs/uuid v3.3.0+incompatible
 	github.com/golang/mock v1.6.0
+	github.com/google/uuid v1.1.1 // indirect
```

Comment:
check f this is necessary

Reply:
Looks like it is. It's a dependency from mastermind.